ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot

```diff
@@ -95,8 +95,19 @@ End To End Pipeline Workflow Using Kfp
     IF    ${ray} == ${TRUE}
         Setup Kueue Resources    ${project}    cluster-queue-user    resource-flavor-user    local-queue-user
     END
-    ${run_id}    Create Run From Pipeline Func    ${username}    ${password}    ${project}
-    ...    ${python_file}    ${method_name}    pipeline_params=${pipeline_params}
+    # the run_robot_test.sh is sending the --variablefile ${TEST_VARIABLES_FILE}
+    # there is no way to double-check if a the key is presented there
+    # the workaround is the following
+    # https://issues.redhat.com/browse/RHOAIENG-6376
```

Comment:
```suggestion
    # The run_robot_test.sh is sending the --variablefile ${TEST_VARIABLES_FILE}
    # which may contain the `PIP_INDEX_URL` and `PIP_TRUSTED_HOST` variables,
    # e.g. for disconnected testing. In such case we need to pass them to the
    # `Create Run From Pipeline Func` so that it knows when to amend the pipeline
    # code for the https://issues.redhat.com/browse/RHOAIENG-6376 issue.
```